resources/helm-broker/values.yaml

```diff
@@ -25,10 +25,10 @@ ctrl:
 broker:
   resources:
     limits:
-      memory: 76Mi
+      memory: 120Mi
```

Comment:
It's much but it makes it more stable for the release, we should investigate the memory usage on grafana.